### ISSUE TITLE
Kee Alpha 5 not working on Nightly

### ISSUE THREAD

**Issue by Dimas-sc** [kee-org/browser-addon#29]
I'm trying Alpha 5 since a week ago. I can't get it working. Normally the icon was detecting KeePass but it didn't detect any form in any page. If I click it i get this: https://i.imgur.com/2LUThR6.png

Today the icon was showing an OFF tooltip until I restarted Firefox, now the icon is good but it does nothing like before.

Please close this issue if all these are the expected results for this alpha. If not I can do some debugging/testing if you need it.

Thx for the work

**Comment by gojpdchx** [comment]
You way wish to take a look at [this comment](https://github.com/kee-org/browser-addon/issues/22#issuecomment-327235719) by the developer regarding the "not detecting any form" issue.

**Comment by luckyrat** [comment]
Alpha 6 should be working in FF 57 with the exception of the issues already described on GitHub. As @gojpdchx points out, there is an outstanding problem with tabs that existed at startup/install time not working reliably but we're waiting on Firefox for the fix for that issue. Anything else you come across would be well worth investigating and reporting here.

**Comment by Dimas-sc** [comment]
I've a new/clean nightly profile. I installed the Alpha 6. It detects KeePass (OFF message dissapears) but no more. It doesn't detect any web form and the popup is always like this: https://i.imgur.com/O2KPeoG.png

**Comment by Dimas-sc** [comment]
Version 57.0a1 20170920100426

**Comment by eikaramba** [comment]
i'm on 57.0b1, and it works. I miss the possibility to use shortkeys to login or switch db, but apart from that, it works ;)

**Comment by hackel** [comment]
Not working at all for me on 57.0b1.  I get a toolbar icon, and clicking it just shows "Loading...".

**Comment by wbedard** [comment]
I also set up FF57b1 with a new profile and Kee 2.0.0.6.  Connecting it to my KeePass instance was uneventful, I plugged in a saved URL in the browser and it did Kee did show the available entries in the password field (username was pre-filled...).  The only unusual thing I noted was that, once I was logged in, Kee offered to save the entry again, even though it populated the login fields from a saved entry.  I'm sure there are several variables that affect this and I only did one test.

My next step is to duplicate my "daily-driver" profile and see how it runs under FF57 beta.  I've converted over (in some cases quite painfully...) to an all-WE lineup for my extensions and it's time to see where the rough edges are going to be once FF57 reaches mainstream.

**Comment by vincekd** [comment]
I'm also getting the "Loading..." screen on FF 57.0b3 64-bit. When I load the extension I get these errors:

```
Reading manifest: Error processing background.persistent: Event pages are not currently supported. This will run as a persistent background page.
Reading manifest: Error processing version_name: An unexpected property was found in the WebExtension manifest.```

**Comment by vincekd** [comment]
It seems gulp isn't properly compiling. It would be convenient for tester if there were pre-compiled js/css provided :).

**Comment by luckyrat** [comment]
@vincekd try:
`gulp build`
then if you want to create an xpi or zip package:
`gulp package:beta`

I've only tested in VSCode recently but it works for me. Let me know if you have any specific errors or problems and I'll try to help further.

The errors when loading the extension are not actual errors - the first is a Firefox bug and the 2nd should be a warning at most - neither have any impact on the extension functionality.

**Comment by Dimas-sc** [comment]
Updated to Alpha 8 and the same happens.

**Comment by vincekd** [comment]
I had to alter the tslint settings to allow `LF` instead of `CRLF` (on linux), and then for some reason gulp was dumping the output js files into the project root rather than their respective folders. Also it couldn't find any of the typescript definition files for some reason, but that didn't affect that output. After that I have it starting/loading the db.

**Comment by dertuxmalwieder** [comment]
I confirm this issue with Alpha 8 and Firefox 57 Beta 4.

**Comment by luckyrat** [comment]
I've split the "Loading..." popup issue into a separate GitHub issue - #32 

If that is the problem you are having, please use that ticket.

I'm actually not even sure what this current issue is supposed to be describing and suspect there may be more independent issues buried within - "it doesn't work" is far too broad to be actionable.

If you're not experiencing the issue in #32, please provide more information about the actual issue, your environment, Kee options enabled, steps to reproduce the problem and any relevant debug logs.

**Comment by bdurrer** [comment]
KeeFox didn't work on the German version for me, but the en-US (aka default) version worked

**Comment by luckyrat** [comment]
Closing since it seems this issue went away and there's not enough specific information to act upon anyway. Feel free to open a new issue if you feel anything in particular from here remains relevant.
